src/ui/mainwindow.cpp

```diff
@@ -407,6 +407,8 @@ void MainWindow::queryCompleted()
     m_treeViewClicked = true;
     ui->treeView->setCurrentIndex(m_searchState->zealSearch->index(0, 0, QModelIndex()));
     ui->treeView->activated(ui->treeView->currentIndex());
+    ui->lineEdit->setFocus();
```

Comment:
Use `Qt::MouseFocusReason` to avoid the need in `deselect()` below. Also this code is only relevant for Qt WebEngine, so a check for `USE_WEBENGINE` should be here. The following code works fine for me. Please check for yourself, and update the pull request.

```cpp
#ifdef USE_WEBENGINE
    ui->lineEdit->setFocus(Qt::MouseFocusReason);
#endif
```